docs/disconnected-updateservice-operator.md

```diff
@@ -108,9 +108,9 @@ container in the updateservice pod:
 wget https://github.com/openshift/cincinnati-graph-data/archive/master.tar.gz -O graph.tar.gz
```

Comment:
You can drop this too, if you're moving to pulling the data from inside the `Dockerfile`.